docs/semgrep-supply-chain/sbom.md

```diff
@@ -3,55 +3,59 @@ slug: /semgrep-supply-chain/sbom
+
+- The vulnerability information from the default branch for the project
+- The dependency information from the latest full scan for the project.
 
 ## Supported standards and formats
 
-Semgrep Supply Chain supports the following:
+Semgrep Supply Chain supports the following SBOM formats:
 
 - CycloneDX 1.4 JSON
 - CycloneDX 1.4 XML
 
-## Generating and downloading an SBOM for a single project
-
-:::info Prerequisites
-- **SBOM generation** can be performed only through Semgrep AppSec Platform.
-- You need at least one successful Supply Chain scan on the **trunk** branch of each repository you want to generate an SBOM for. See [<i class="fa-regular fa-file-lines"></i> Core deployment](/deployment/core-deployment) to set up your Semgrep account and Supply Chain scans.
-:::
-
-When generating an SBOM, Semgrep uses the vulnerability information from the default branch for the project, and the dependency information from the latest full scan for the project. Typically, full scans are run only on default branches, but if your workflow differs and you also run full scans on other branches, that can create a mismatch between dependencies and vulnerabilities in the generated SBOM.
+## Generate and download an SBOM for a single project
 
-To avoid the mismatch, ensure that the latest full scan ran on the default branch of the repository you want to generate an SBOM for.
+SBOM generation can be performed only through Semgrep AppSec Platform.
 
-1. In Semgrep AppSec Platform, click **Supply Chain > Dependencies**.
+1. In Semgrep AppSec Platform, go to **Supply Chain > Dependencies**.
 2. Click the **Download <i class="fa-solid fa-download"></i>** icon next to the repository you want an SBOM for.
-3. Click the format you want the SBOM to be in. After clicking, do not refresh or leave the page until the SBOM has been generated.
-3. Once Semgrep has generated the SBOM, click the link provided on the toaster notification to download it.
-    ![Download link in toaster notification](/img/download-sbom.png) _**Figure**. The Dependencies tab displaying a successful SBOM download._
+3. Click the format you want the SBOM to be in. After clicking, refresh or leave the page only after the SBOM has been generated.
+4. Once Semgrep has generated the SBOM, click the link on the toaster notification to download it.
+    ![Download link in toaster notification](/img/download-sbom.png) _**Figure**. The Dependencies tab displays a successful SBOM download._
 
 You have successfully downloaded an SBOM.
 
+:::note Supply Chain scans on non-primary branches
+Typically, full scans are run only on default branches. However, if your workflow differs and you run full scans on non-primary branches, this can create a mismatch between dependencies and vulnerabilities in the generated SBOM. To avoid the mismatch, ensure that the latest full scan runs on the default branch of the repository for which you want to generate an SBOM.
```

Comment:
```suggestion
Typically, full scans are run only on primary (default) branches. However, if your workflow differs and you run full scans on non-primary branches, this can create a mismatch between dependencies and vulnerabilities in the generated SBOM. To avoid the mismatch, ensure that the latest full scan runs on the primary branch of the repository for which you want to generate an SBOM.
```